app/lib/screens/study/tasks/observation/questionnaire_task_widget.dart

```diff
@@ -65,7 +65,7 @@ class _QuestionnaireTaskWidgetState extends State<QuestionnaireTaskWidget> {
                   _isLoading = true;
                 });
                 switch (response.runtimeType) {
-                  case QuestionnaireState _:
+                  case == QuestionnaireState:
```

Comment:
We should avoid using `runtimeType` as it can lead to unwanted behavior.

See: https://stackoverflow.com/a/42802802
See: https://github.com/dart-lang/site-www/issues/3354
See (only a problem if obfuscation is enabled): https://docs.flutter.dev/deployment/obfuscate#caveat

With Dart 3.0 we can use pattern matching in switch statements like:

`case QuestionnaireState():
`

OR

`case QuestionnaireState _:
`